cv32e40p_manifest.flist

```diff
@@ -57,7 +59,7 @@ ${DESIGN_RTL_DIR}/cv32e40p_prefetch_controller.sv
 ${DESIGN_RTL_DIR}/cv32e40p_sleep_unit.sv
 ${DESIGN_RTL_DIR}/cv32e40p_core.sv
 ${DESIGN_RTL_DIR}/../bhv/cv32e40p_sim_clock_gate.sv
-${DESIGN_RTL_DIR}/../bhv/cv32e40p_tracer.sv
 ${DESIGN_RTL_DIR}/../bhv/cv32e40p_wrapper.sv
-${DESIGN_RTL_DIR}/../bhv/cv32e40p_core_log.sv
-${DESIGN_RTL_DIR}/../bhv/cv32e40p_apu_tracer.sv
\ No newline at end of file
+//${DESIGN_RTL_DIR}/../bhv/cv32e40p_tracer.sv
```

Comment:
Why are these 3 files commented out; you will need them in core-v-verif

Reply:
This was done while trying to "put the binds as high as possible" so that there would only be one set of compiler macros to control compilation and binding of the tracer and assertions.  Verilator cannot handle the tracer, assertions, etc, so if these modules cannot have conditional compile macros, then they cannot be in the manifest.

The proposed solution is to place conditionally compiled tick-includes in the wrapper.

Reply:
My point is that you still need these files to be loaded/compiled in core-v-verif, so if you do not put them in the manifest your code in core-v-verif won't compile.

Reply:
That is correct for the UVM environment.    The Makefles know this and set the appropriate macros that compile in the tracer.  Take a look at the top of cv32e40p_wrapper.sv to see what I mean.   Verilator cannot parse the tracer, so the Makefiles for the core testbench do not use these macros.

Reply:
I don't get it, but that's fine. I think you will simply be missing files in core-v-verif if you try to compile for UVM. (Anyway, you can leave them commented out).

Reply:
The inclusions are at the top of `cv32e40p_wrapper.sv` _not_ in the manifest:
```
`ifdef CV32E40P_SVASSERTIONS
  `include "cv32e40p_prefetch_controller_sva.sv"
`endif

`include "cv32e40p_core_log.sv"

`ifdef CV32E40P_APU_TRACE
  `include "cv32e40p_apu_tracer.sv"
`endif

`ifdef CV32E40P_TRACE_EXECUTION
  `include "cv32e40p_tracer.sv"
`endif
```

Reply:
Okay, now I get it. That is quite 'non-conventional'

Reply:
I don't like it much myself.  Once we get a proper manifest generator tool in place we can get rid of this.  The single benefit is that the compiler macros (the tick-ifdefs) are all in one file.